packages/web/app/views/patients/components/FinalisePatientMoveModal.jsx

```diff
@@ -2,62 +2,69 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import styled from 'styled-components';
 import { Box } from '@material-ui/core';
-import SingleBedIcon from '@material-ui/icons/SingleBed';
-import { Modal, TranslatedText } from '@tamanu/ui-components';
+import { BedIcon } from '../../../assets/icons/BedIcon';
+import { Modal, TAMANU_COLORS, TranslatedText } from '@tamanu/ui-components';
 import { Colors } from '../../../constants/styles';
 import { BodyText } from '../../../components';
 import { ModalActionRow } from '../../../components/ModalActionRow';
 import { usePatientMove } from '../../../api/mutations';
 import { getFullLocationName } from '../../../utils/location';
 
 const Text = styled(BodyText)`
-  color: ${props => props.theme.palette.text.secondary};
   margin-top: 10px;
   margin-bottom: 40px;
 `;
 
 const Container = styled.div`
   display: flex;
   margin: 40px auto 60px;
-  padding: 0 30px;
+  justify-content: center;
 `;
 
-const BedIcon = styled(SingleBedIcon)`
-  color: ${Colors.softText};
-  font-size: 50px;
-
-  &.MuiSvgIcon-colorPrimary {
-    color: ${props => props.theme.palette.primary.main};
-  }
+const Dots = styled.div`
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  margin-left: 12px;
 `;
 
 const Dot = styled.div`
   display: inline-block;
   height: 5px;
   width: 5px;
   margin: 10px 0;
-  background: ${Colors.darkText};
+  background: ${Colors.primary};
```

Comment:
```suggestion
  background: ${({ theme }) => theme.palette.primary.main};
```
It's very minor but I reckon it's best to use the theme when possible for the colors as it makes refactoring easier.